src/components/DayPicker.jsx

```diff
@@ -260,13 +260,20 @@ export default class DayPicker extends React.Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
+    const { orientation, daySize } = this.props;
     const { monthTransition, currentMonth, focusedDate } = this.state;
     if (monthTransition || !currentMonth.isSame(prevState.currentMonth)) {
       if (this.isHorizontal()) {
         this.adjustDayPickerHeight();
       }
     }
 
+    if (orientation !== prevProps.orentation || daySize !== prevProps.daySize) {
```

Comment:
Not super sure why this isn't just integrated into the above conditional? Let's make that change and then merge this in.